Author: Timo Aaltonen <tjaalton@debian.org>
Date:   Fri Feb 8 19:06:19 2019 +0200

Revert "i965: Allocate VMA in userspace for full-PPGTT systems."
    
This reverts commit a363bb2cd0e2a141f2c60be005009703bffcbe4e.

softpin is still buggy with current 4.15 and 4.18 kernels.
https://bugs.launchpad.net/ubuntu/cosmic/+source/mesa/+bug/1815172
https://bugs.launchpad.net/ubuntu/+source/mesa/+bug/1836721

diff --git a/src/mesa/drivers/dri/i965/brw_bufmgr.c b/src/mesa/drivers/dri/i965/brw_bufmgr.c
index 19e2d145f4a..4b30602d3ef 100644
--- a/src/mesa/drivers/dri/i965/brw_bufmgr.c
+++ b/src/mesa/drivers/dri/i965/brw_bufmgr.c
@@ -1723,7 +1723,7 @@ brw_bufmgr_init(struct gen_device_info *devinfo, int fd)
       bufmgr->initial_kflags |= EXEC_OBJECT_SUPPORTS_48B_ADDRESS;
 
       /* Allocate VMA in userspace if we have softpin and full PPGTT. */
-      if (gem_param(fd, I915_PARAM_HAS_EXEC_SOFTPIN) > 0 &&
+      if (false && gem_param(fd, I915_PARAM_HAS_EXEC_SOFTPIN) > 0 &&
           gem_param(fd, I915_PARAM_HAS_ALIASING_PPGTT) > 1) {
          bufmgr->initial_kflags |= EXEC_OBJECT_PINNED;
 
